src/panels/custom/ha-panel-custom.ts

```diff
@@ -78,13 +78,23 @@ export class HaPanelCustom extends UpdatingElement {
       !["localhost", "127.0.0.1", location.hostname].includes(tempA.hostname)
     ) {
       if (
-        !confirm(`Do you trust the external panel "${config.name}" at "${
-          tempA.href
-        }"?
-
-It will have access to all data in Home Assistant.
-
-(Check docs for the panel_custom component to hide this message)`)
+        !confirm(
+          this.hass.localize(
+            "ui.panel.custom.external_panel.question_trust",
+            "name",
+            config.name,
+            "link",
+            tempA.href
+          ) +
+            "\n\n" +
+            this.hass.localize(
+              "ui.panel.custom.external_panel.complete_access"
+            ) +
+            "\n\n" +
+            "(" +
+            this.hass.localize("ui.panel.custom.external_panel.hide_message") +
+            ")"
```

Comment:
Use a template literal instead of concatting strings.

Reply:
Resolved: https://github.com/home-assistant/home-assistant-polymer/pull/4137/commits/4b6810be37ef69730daed9a9ffb1c4046b2be5a2